burn-compute/src/tune/tuner.rs

```diff
@@ -62,16 +62,20 @@ impl<S: ComputeServer, C: ComputeChannel<S>> Tuner<S, C> {
             })
             .collect();
 
-        for (name, result) in names.iter().zip(results.iter()) {
-            log::info!("Benchmark result {name}-{key} => {result}");
-        }
-
         // Finds the fastest operation, stores it and returns it
         let fastest_index = self.find_fastest(results);
         let fastest_name = names.get(fastest_index).unwrap();
         log::info!("Fastest result {fastest_name}-{key}");
 
-        self.tune_cache.cache_insert(key, fastest_index);
+        self.tune_cache.cache_insert(key.clone(), fastest_index);
+        #[cfg(feature = "autotune-persistent-cache")]
+        {
+            let checksum = autotune_operation_set.compute_checksum();
+            self.tune_cache
+                .persistent_cache_insert(key, checksum, fastest_index);
+            self.tune_cache.save();
```

Comment:
I think it would be better to avoid writing to disk each time we auto-tune with persistent cache enabled. It's not a priority in this PR, but if you can implement the `Drop` trait for `Tuner` and call `self.tune_cache.save()`, it would save the cache when the tuner is deconstructed.

Reply:
For reference, we tried to do it but COMPUTE is static and thus it is never dropped. We decided to keep saving the cache each time an operation set is autotuned.